access-control-app/src/cmr/access_control/api/routes.clj

```diff
@@ -380,8 +380,20 @@
       (context "/current-sids" []
         (OPTIONS "/" [] (common-routes/options-response))
 
-        (GET "/" {:keys [request-context params]}
-             (get-current-sids request-context params)))
+        (if (access-control-config/enable-sids-get)
+          (GET "/"
+               {:keys [request-context params]}
+               (do
+                 (error (format "client [%s] Using GET instead of POST when requesting current sids"
+                                (:client-id request-context)))
+                 (get-current-sids request-context params)))
+          {:status 404})
+
+        (POST "/"
+              {:keys [request-context body]}
+              (get-current-sids request-context (-> (slurp body)
+                                                    (json/parse-string true)
+                                                    (clojure.walk/keywordize-keys)))))
```

Comment:
nit;
```suggestion
                                                    (walk/keywordize-keys)))))
```

Reply:
Then add `[clojure.walk :as walk]` as dependency

Reply:
you know, I was not convinced that was going to be the solution so I didn't bother to add it, then I forgot. I ended up removing it.